### ISSUE TITLE
Nightly build against master failing

### ISSUE THREAD

**Issue by jrudolph** [akka/akka-http#1144]
First failure: https://jenkins.akka.io:8498/view/Akka-http/job/akka-http-nightly-against-akka-master/164/

Some build problem (sbt plugin incompatibility?):

```
[error] /home/jenkins/.sbt/0.13/staging/8bd016127453b4b8fefa/akka/project/Version.scala:15: No implicit for Append.Value[Seq[sbt.Task[Seq[java.io.File]]], sbt.Def.Initialize[sbt.Task[Seq[sbt.File]]]] found,
[error]   so sbt.Def.Initialize[sbt.Task[Seq[sbt.File]]] cannot be appended to Seq[sbt.Task[Seq[java.io.File]]]
[error]     resourceGenerators += generateVersion(resourceManaged, _ / "version.conf",
[error]                        ^
[error] /home/jenkins/.sbt/0.13/staging/8bd016127453b4b8fefa/akka/project/Version.scala:18: No implicit for Append.Value[Seq[sbt.Task[Seq[java.io.File]]], sbt.Def.Initialize[sbt.Task[Seq[sbt.File]]]] found,
[error]   so sbt.Def.Initialize[sbt.Task[Seq[sbt.File]]] cannot be appended to Seq[sbt.Task[Seq[java.io.File]]]
[error]     sourceGenerators += generateVersion(sourceManaged, _ / "akka" / "Version.scala",
[error]                      ^
[error] two errors found
[error] (compile:compileIncremental) Compilation failed
```

**Comment by jrudolph** [comment]
Seems to be caused by using a [feature](http://developer.lightbend.com/blog/2017-04-10-sbt-01315-JDK9-support-and-offline-installation/#build-sbt-dsl-improvements) that was added in sbt 0.13.15. As akka-http was still on sbt 0.13.13, in the combined build, sbt version was downgraded to 0.13.13 also for the akka build.

**Comment by jrudolph** [comment]
Fixed in #1145.
